src/components/PopupWithForm.js

```diff
@@ -14,26 +24,42 @@ class PopupWithForm extends Popup {
     inputs.forEach((input) => {
       values[input.name] = input.value;
     });
+
     return values;
   }
 
   _submitForm = () => {
     const inputValues = this._getInputValues();
-    this._handleFormSubmit(inputValues);
+    this._loadingState = true;
+    this.showLoading(true);
```

Comment:
Please remove this, you should show the loading in index.js before the request

Reply:
Start the loading in index.js before the request

Reply:
Needs correction: Start the loading in index.js before the request, not here

The api call should look like this:

```js
popup.showLoading()
api.
  .then(...)
  .catch(...)
  .finally(() => {
    popup.hideLoading()

  })
```

Reply:
Needs correction: Start the loading in index.js before the request, not here

Reply:
Needs correction: Start the loading in index.js before the request, not here